sipn_reanalysis_ingest/luigitasks/convert.py

```diff
@@ -4,20 +4,26 @@
 import luigi
 
 from sipn_reanalysis_ingest._types import CfsrGranuleProductType
-from sipn_reanalysis_ingest.constants.cfsr import CFSR_VERSION_BY_DATE
+from sipn_reanalysis_ingest.constants.cfsr import (
+    CFSR_VERSION_BY_DATE,
+    CFSR_DAILY_TAR_BEFORE
+)
 from sipn_reanalysis_ingest.constants.paths import (
     DATA_DAILY_FILENAME_TEMPLATE,
     DATA_FINISHED_DIR,
     DATA_MONTHLY_FILENAME_TEMPLATE,
 )
 from sipn_reanalysis_ingest.luigitasks.untar import (
+    UntarCfsr1DayFile,
     UntarCfsr5DayFile,
     UntarCfsrV1MonthlyFile,
     UntarCfsrV2MonthlyFile,
 )
 from sipn_reanalysis_ingest.util.cfsr import (
-    select_6hourly_analysis_grib2s,
-    select_6hourly_forecast_grib2s,
+    select_v1_6hourly_analysis_grib2s,
+    select_v1_6hourly_forecast_grib2s,
+    select_v2_6hourly_analysis_grib2s,
+    select_v2_6hourly_forecast_grib2s,
```

Comment:
I'd like to avoid v1/v2 designation here, because we'll actually be calling the v1 function for v2 data on e.g. January 1st 2011. What about `select_daily_6hourly_analysis_grib2s` and `select_5daily_analysis_grib2s` instead? Having two "periodicities" in one filename isn't great, but we could write a docstring for those functions that clarifies.